website_event_require_login/views/website_event_templates.xml

```diff
@@ -1,33 +1,39 @@
 <?xml version="1.0" encoding="utf-8" ?>
 <odoo>
     <template id="modal_attendees_registration_login_required" name="Login Required">
-        <div id="modal_attendees_registration_login_required" class="modal fade">
-            <div class="modal-dialog">
+        <div
+            id="modal_attendees_registration_login_required"
+            class="modal fade"
+            tabindex="-1"
+            role="dialog"
+        >
+            <div class="modal-dialog modal-lg">
                 <div class="modal-content">
-                    <div class="modal-header">
+                    <div class="modal-header align-items-center">
+                        <h4 class="modal-title">Login required</h4>
                         <button
                             type="button"
-                            class="close"
-                            data-dismiss="modal"
+                            class="btn-close"
+                            data-bs-dismiss="modal"
                             aria-hidden="true"
-                        >&amp;times;</button>
-                        <h4 class="modal-title">Login required</h4>
+                            aria-lable="Close"
```

Comment:
```suggestion
                            aria-label="Close"
```

Reply:
Thanks for you review @pedrobaeza I missed that one !